lib/iris/tests/unit/fileformats/dot/test__dot_path.py

```diff
@@ -0,0 +1,51 @@
+# (C) British Crown Copyright 2016, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit tests for :func:`iris.fileformats.dot._dot_path`."""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import os.path
+import subprocess
+
+import mock
+
+from iris.fileformats.dot import _dot_path
+
+
+class Test(tests.IrisTest):
+    def test_broken_install(self):
```

Comment:
Might we also want to test this when it *does* have a valid path ?
Mind you, it must work because the dot functionality itself is tested elsewhere (```tests.test_cdm``` and ```tests.test_file_save```)